api/base/types.go

```diff
@@ -125,7 +125,6 @@ type UserModelSummary struct {
 	ControllerUUID     string
 	IsController       bool
 	ProviderType       string
-	DefaultSeries      string
```

Comment:
Should we return `DefaultBase` instead?

Reply:
No need really imo. We won't use it anywhere